concurrent/src/main/scala/tofu/concurrent/impl/contextt.scala

```diff
@@ -319,8 +319,7 @@ final class ContextTRunContext[F[+_]: Applicative, C[_[_]]](implicit FD: Defer[F
 
 // instance that does not defer locals. could be stack-unsafe
 final class ContextTRunContextUnsafe[F[+_]: Applicative, C[_[_]]]
-    extends ContextTContext[F, C] with RunContext[ContextT[F, C, *]]                       {
-  type Lower[+A] = F[A]
+    extends ContextTContext[F, C] with WithRun[ContextT[F, C, *], F, C[ContextT[F, C, *]]] {
```

Comment:
Why exactly do we need it?

Reply:
To drop Defer requirement when it's supposed to be fine